### ISSUE TITLE
Display tags always (not just as tooltip) for each hit in the left side.

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#741]

**Comment by phraenquex** [comment]
Fix as part of #540

**Comment by phraenquex** [comment]
This should include fixing the "assign tags" modal:  the tags should behave just like in the "reworked" Tag Details LHS panel.

Ideally, make it switch between list and grid views too (like #1102).

In fact, might be simplest to simply use the Tag Details pane as is (without buttons), and just change what happens when you click on a tag.

Currently (bleurgh): 
![image](https://github.com/m2ms/fragalysis-frontend/assets/33267937/32c09f5f-a4f3-4ff8-8720-8ccb8c093ebf)

Should be more like this: 
![image](https://github.com/m2ms/fragalysis-frontend/assets/33267937/214dcdd4-a601-4de3-8952-f6c29a19c076)

**Comment by phraenquex** [comment]
Um... maybe.  The tags should be very small - try just the first 2-3 letters, and the colour.  Then the tag generator can name the tags in a way that they're useful in the small format.  

(Think how Gmail and Outlook do with senders on a smartphone.)

**Comment by boriskovar-m2ms** [comment]
Yes that's how it's implemented and they're also slightly transparent. In few (hopefully) moments you'll see it deployed on tibor's stack.

**Comment by phraenquex** [comment]
Below is is what I had in mind.  
![image](https://github.com/m2ms/fragalysis-frontend/assets/33267937/01348f3c-25fd-4ed2-abf4-c71254bb5086)

I do like the hover-blow-out you do, mind you, so keep that, BUT: 
* reuse the formatting from the Tag Details panel
* (if easy) order them by the same order they have in the Tag Details panel

**Comment by RoboMatuska** [comment]
@phraenquex 

I have an additional question:
What is the maximum number tags columns and rows for one compound? or you want show a preview all selected tags?

**Comment by phraenquex** [comment]
Good question - which reminds me, there's another problem.  I'll answer both.

1. 
The tags should be **next to ** the 2D molecule image, not over it.  
* this is because the 2D molecule image _also_ has a mouse-hover action - and currently it's annoying to have to figure out where to put the mouse for either the enlarged 2D or the tag blow-up.
* shrink the width of the 2D molecule image accordingly.

2.  
For the small tags, make space for 2 columns with 5 rows (I doubt any compound will have more than 10 tags)

3. 
For the blow-out view, I think the grid layout from the Details Pane would work.  (I'd need to see it though to know.)
